src/query/server/server_test.go

```diff
@@ -47,10 +47,12 @@ import (
 	"google.golang.org/grpc"
 )
 
+var queryPort = 25123
+
 var configYAML = `
 listenAddress:
   type: "config"
-  value: "127.0.0.1:7201"
+  value: "127.0.0.1:25123"
```

Comment:
Might be worth templating with sprintf? Not really a concern since it's a test

Reply:
Thought about it, but I figured it'd be more difficult to look for the %d's in a long config string than to just have the hardcoding.